idaes/generic_models/properties/core/eos/tests/test_enrtl_verification_2_solvent.py

```diff
@@ -0,0 +1,359 @@
+from idaes.generic_models.properties.core.pure.electrolyte import \
+    relative_permittivity_constant
+
+
+def rho_H2O(b, *args, **kwargs):
+    return 1000/18e-3*pyunits.mol/pyunits.m**3
+
+
+def rho_MeOH(b, *args, **kwargs):
+    return 792/32e-3*pyunits.mol/pyunits.m**3
+
+
+def rho_EtOH(b, *args, **kwargs):
+    return 789.45/46e-3*pyunits.mol/pyunits.m**3
+
+
+configuration = {
+    "components": {
+        "H2O": {"type": Solvent,
+                "dens_mol_liq_comp": rho_H2O,
+                "relative_permittivity_liq_comp":
+                    relative_permittivity_constant,
+                "parameter_data": {
+                    "mw": (18E-3, pyunits.kg/pyunits.mol),
+                    "relative_permittivity_liq_comp": 78.54}},
+        "MeOH": {"type": Solvent,
+                 "dens_mol_liq_comp": rho_MeOH,
+                 "relative_permittivity_liq_comp":
+                     relative_permittivity_constant,
+                 "parameter_data": {
+                     "mw": (32E-3, pyunits.kg/pyunits.mol),
+                     "relative_permittivity_liq_comp": 32.6146}},
+        "EtOH": {"type": Solvent,
+                 "dens_mol_liq_comp": rho_EtOH,
+                 "relative_permittivity_liq_comp":
+                     relative_permittivity_constant,
+                 "parameter_data": {
+                     "mw": (46E-3, pyunits.kg/pyunits.mol),
+                     "relative_permittivity_liq_comp": 24.113}},
+        "NaBr": {"type": Apparent,
+                 "dissociation_species": {"Na+": 1, "Br-": 1}},
+        "Na+": {"type": Cation,
+                "charge": +1},
+        "Br-": {"type": Anion,
+                "charge": -1}},
+    "phases": {
+        "Liq": {"type": AqueousPhase,
+                "equation_of_state": ENRTL}},
+    "base_units": {"time": pyunits.s,
+                   "length": pyunits.m,
+                   "mass": pyunits.kg,
+                   "amount": pyunits.mol,
+                   "temperature": pyunits.K},
+    "state_definition": FTPx,
+    "state_components": StateIndex.true,
+    "pressure_ref": 1e5,
+    "temperature_ref": 300,
+    "parameter_data": {
+        "Liq_alpha": {
+            ("H2O", "EtOH"): 0.3031,
+            ("H2O", "MeOH"): 0.2994,
+            ("MeOH", "EtOH"): 0.3356,
+            ("H2O", "Na+, Br-"): 0.2,
+            ("MeOH", "Na+, Br-"): 0.2,
+            ("EtOH", "Na+, Br-"): 0.1},
+        "Liq_tau": {
+            ("H2O", "MeOH"): 1.4265,
+            ("MeOH", "H2O"): -0.42864,
+            ("H2O", "EtOH"): 2.2485,
+            ("EtOH", "H2O"): -0.18514,
+            ("MeOH", "EtOH"): -0.04394,
+            ("EtOH", "MeOH"): 0.02147,
+            ("H2O", "Na+, Br-"): 9.527,
+            ("Na+, Br-", "H2O"): -4.790,
+            ("MeOH", "Na+, Br-"): 5.910,
+            ("Na+, Br-", "MeOH"): -3.863,
+            ("EtOH", "Na+, Br-"): 6.118,
+            ("Na+, Br-", "EtOH"): -4.450}}}
+
```

Comment:
I assume these data are correct but wanted to do a quick double-check. Can you direct me to where I can find these values or were they digitized from a figure in the reference?

Reply:
All of these come from table 3 in the 2009 paper.

Reply:
Should `"temperature_ref": 300` be set to 298.15 instead? It seems like it won't make much of a difference, but is there a particular reason why `"temperature_ref"` is 300 instead of 298.15?

Reply:
For this it doesn't matter - Tref is not used in the activity coefficient calculations. It would be important if we were testing things like enthalpy, and then it should be matched to the Href being used.